src/pages/index.js

```diff
@@ -45,30 +73,68 @@ const section = new Section(
 );
 section.renderItems();
 
-const userInfo = new UserInfo({
-  nameSelector: ".profile__title",
-  descriptionSelector: ".profile__description",
+// retrieve use info
+api.getUserInfo().then((data) => {
+  // console.log(data);
+  userInfo.setUserInfo({
+    name: data.name,
+    about: data.about,
+    avatar: data.avatar,
+  });
 });
 
+function handleProfileFormSubmit(inputValues) {
+  userInfo.setUserInfo({
+    name: inputValues.name,
+    about: inputValues.about,
+    avatar: inputValues.avatar,
+  });
+
+  return api
+    .setUserInfo({
+      name: inputValues.name,
+      about: inputValues.about,
+      avatar: inputValues.avatar,
+    })
+    .then((data) => {
+      // after updating, reflect the changes in the UI
+      userInfo.setUserInfo({
+        name: data.name,
+        about: data.about,
+        avatar: data.avatar,
+      });
+      profilePopup.close();
```

Comment:
You close the modal inside `PopupWithForm` already in `then`, so you don't need it in `index.js`